backend/populateDB/schema.sql

```diff
@@ -664,6 +668,37 @@ $$ language sql stable security definer;
 
 comment on function floods.crossing_communities(floods.crossing) is 'Get all the communities for a crossing.';
 
+create function floods.legacy_jurisdiction_abbreviation(crossing floods.crossing) returns text as $$
+  select com.abbreviation from floods.community com
+  where array_position(crossing.community_ids, com.id) >= 0
+  limit 1;
+$$ language sql stable security definer;
+
+comment on function floods.legacy_jurisdiction_abbreviation(floods.crossing) is 'Gets the legacy abbreviation of the first community for a crossing.';
+
+-- Create function to generate legacy xml
+create function floods.legacy_xml(
```

Comment:
This is missing the `comment` attribute.

Compare `/xml` to https://atxfloods.com/dashboard/phpsqlajax_genxml.php:

```
<marker
  name="Low Water Crossing #1"
  jurisdiction="COA"
  address="6019 Spicewood Springs, Austin, TX"
  lat="30.390406"
  lng="-97.774994"
  type="on"
  id="1"
/>
```

```diff
<marker
  name="Low Water Crossing #1"
  jurisdiction="COA"
  address="6019 Spicewood Springs, Austin, TX"
  lat="30.390406"
  lng="-97.774994"
  type="on"
+ comment="Crossing is open."
  id="1"
/>
```

Reply:
fixed